docs/migrate-to-ory/auth0.mdx

```diff
@@ -0,0 +1,140 @@
+
+```shell
+export AUTH0_DOMAIN="{your_auth0_domain}.auth0.com"
+export AUTH0_CONNECTION_ID="{your_auth0_connection_id}"
+export AUTH0_TOKEN="{your_auth0_management_api_token}"
+```
+
+```shell
+bash <(curl https://raw.githubusercontent.com/ory/docs/auth0guide/code-examples/migrate-to-ory/0-get-auth0-user-data.sh)
+```
+
+### Creating identities
+
+Once the download is finished and Auth0 provided you with the hashed passwords, you can create the identities
+`in Ory Kratos / Ory Network?`, requiring the users to login with email and password.
+
+Since it's possible your user data contains new users who weren't registered when the hashed passwords were exported, we'll
+reserve an identity for that email. If you're on a free Auth0 plan and you don't qualify for an export of your hashed passwords,
+then this might be a viable option for you (`pw reset flow?`).
+
+Now you create an Ory Project incase you don't have one yet:
+
+`setting schema this isnt pretty - i know, see https://github.com/ory/docs/pull/1157`
+
+```shell
+ory create project --name "Ory Docs Auth0 Migration Example"
+export ORY_PROJECT_ID='{set to the project ID from output}'
+
+# Use the email identity schema preset
+ory patch identity-config $ORY_PROJECT_ID \
+     --replace '/identity/default_schema_id="preset://email"' \
+     --replace '/identity/schemas=[{"id":"preset://email","url":"base64://ewogICIkaWQiOiAiaHR0cHM6Ly9zY2hlbWFzLm9yeS5zaC9wcmVzZXRzL2tyYXRvcy9pZGVudGl0eS5lbWFpbC5zY2hlbWEuanNvbiIsCiAgIiRzY2hlbWEiOiAiaHR0cDovL2pzb24tc2NoZW1hLm9yZy9kcmFmdC0wNy9zY2hlbWEjIiwKICAidGl0bGUiOiAiUGVyc29uIiwKICAidHlwZSI6ICJvYmplY3QiLAogICJwcm9wZXJ0aWVzIjogewogICAgInRyYWl0cyI6IHsKICAgICAgInR5cGUiOiAib2JqZWN0IiwKICAgICAgInByb3BlcnRpZXMiOiB7CiAgICAgICAgImVtYWlsIjogewogICAgICAgICAgInR5cGUiOiAic3RyaW5nIiwKICAgICAgICAgICJmb3JtYXQiOiAiZW1haWwiLAogICAgICAgICAgInRpdGxlIjogIkUtTWFpbCIsCiAgICAgICAgICAib3J5LnNoL2tyYXRvcyI6IHsKICAgICAgICAgICAgImNyZWRlbnRpYWxzIjogewogICAgICAgICAgICAgICJwYXNzd29yZCI6IHsKICAgICAgICAgICAgICAgICJpZGVudGlmaWVyIjogdHJ1ZQogICAgICAgICAgICAgIH0sCiAgICAgICAgICAgICAgIndlYmF1dGhuIjogewogICAgICAgICAgICAgICAgImlkZW50aWZpZXIiOiB0cnVlCiAgICAgICAgICAgICAgfSwKICAgICAgICAgICAgICAidG90cCI6IHsKICAgICAgICAgICAgICAgICJhY2NvdW50X25hbWUiOiB0cnVlCiAgICAgICAgICAgICAgfQogICAgICAgICAgICB9LAogICAgICAgICAgICAicmVjb3ZlcnkiOiB7CiAgICAgICAgICAgICAgInZpYSI6ICJlbWFpbCIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgInZlcmlmaWNhdGlvbiI6IHsKICAgICAgICAgICAgICAidmlhIjogImVtYWlsIgogICAgICAgICAgICB9CiAgICAgICAgICB9LAogICAgICAgICAgIm1heExlbmd0aCI6IDMyMAogICAgICAgIH0KICAgICAgfSwKICAgICAgInJlcXVpcmVkIjogWwogICAgICAgICJlbWFpbCIKICAgICAgXSwKICAgICAgImFkZGl0aW9uYWxQcm9wZXJ0aWVzIjogZmFsc2UKICAgIH0KICB9Cn0K"}]'
+```
+
+Export the paths to your hashed passwords and your Auth0 user data like this:
+
+```shell
+export AUTH0_USERDATA="{/path/to/AUTH0_USERDATA.json}"
+export AUTH0_PWEXPORT="{/path/to/hashed_passwords.json}" # comment this line to reserve not create identities
```

Comment:
`comment this line to reserve not create identities` 
???

Reply:
I've added an extra option to make it less confusing